packages/react-ui/src/widgets/TableWidgetUI/TableWidgetUI.js

```diff
@@ -89,10 +91,16 @@ function TableWidgetUI({
     onSetPage(0);
   };
 
+  const fixedHeightStyle = {};
+  if (height) {
+    const paginationHeight = paginationRef?.current?.clientHeight || 0;
+    fixedHeightStyle.height = `calc(${height} - ${paginationHeight}px)`;
+  }
```

Comment:
What do you think about using a useMemo here? Idk if it's useful, but I find weird to see this kind of logic being executed in every render.